WordPress/Classes/ViewRelated/System/NetworkAware.swift

```diff
@@ -22,7 +22,7 @@ extension NetworkAware {
 
 extension NetworkAwareUI {
```

Comment:
Instead of conforming the protocol to UIViewController can we specify on the extension:
`extension NetworkAwareUI where Self: UIViewController`
?

Reply:
🤦‍♂ That is so much better. Thank you for spotting that! Fixed in 8132aa3bf7d61ae0408e85a3ea46ac1909a1ed31.